requirements/edx/base.txt

```diff
@@ -48,7 +48,7 @@ edx-lint==0.4.3
 astroid==1.3.8
 edx-django-oauth2-provider==1.1.4
 edx-django-sites-extensions==2.3.0
-edx-enterprise==0.39.9
+edx-enterprise==0.40.0
```

Comment:
@brittneyexline Yeah I was planning on maybe making a separate PR for it just in case something in this one breaks something on stage. I'm probably just being over precautious, though.